src/main/java/com/xiaomi/infra/pegasus/example/BatchSample.java

```diff
@@ -0,0 +1,104 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package com.xiaomi.infra.pegasus.example;
+
+import com.xiaomi.infra.pegasus.client.PException;
+import com.xiaomi.infra.pegasus.client.PegasusClientFactory;
+import com.xiaomi.infra.pegasus.client.PegasusClientInterface;
+import com.xiaomi.infra.pegasus.client.PegasusTableInterface;
+import com.xiaomi.infra.pegasus.client.request.BatchWithResponse;
+import com.xiaomi.infra.pegasus.client.request.Delete;
+import com.xiaomi.infra.pegasus.client.request.DeleteBatch;
+import com.xiaomi.infra.pegasus.client.request.Get;
+import com.xiaomi.infra.pegasus.client.request.GetBatch;
+import com.xiaomi.infra.pegasus.client.request.Set;
+import com.xiaomi.infra.pegasus.client.request.SetBatch;
+import io.netty.util.concurrent.Future;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+
+public class BatchSample {
+
+  public void batch() throws PException {
+    String tableName = "temp";
+    PegasusClientInterface client = PegasusClientFactory.getSingletonClient();
+    PegasusTableInterface table = client.openTable(tableName);
+
+    List<Set> sets = new ArrayList<>();
+    sets.add(new Set("hashKeySet_1".getBytes(), "sortKeySet1".getBytes(), "valueSet1".getBytes()));
+    sets.add(
+        new Set("hashKeySet_2".getBytes(), "sortKeySet2".getBytes(), "valueSet2".getBytes())
+            .withTTLSeconds(1000));
+    new SetBatch(table, 1000).commit(sets);
```

Comment:
After I read your example, I come up with an idea: what if we can support API like this:

```
SetBatch sets = new SetBatch(table, 1000);
sets.add(new Set(...));
sets.commit();
```

User will not need to create one more list. Possible implementation:

```
class AbstractBatch<Request> {
  public void add(Request req) {
     requests.add(req);
  }

  protected List<Request> requests;
}
```

Reply:
* It means that one `SetBatch` instance only be used once, and you have to create new instance if you want to send another batch request.
* sometime, user may not use `sets.add(new Set(...))` to init but pass one List, such as in Spark to load data, so we may not should limit the source data init way

Reply:
> It means that one SetBatch instance only be used once, and you have to create new instance if you want to send another batch request.

Why reuse SetBatch? Do you intend to save java objects? Add a `clear` method to `AbstractBatch` for this purpose.

Reply:
So the `clear` is auto-action after `commit`？if `clear` and `add` exist meanwhile, will it casuse thread-safe question?